.gitignore

```diff
@@ -5,3 +5,5 @@ spec/fixtures
 .vagrant
 .bundle
 vendor
+.ruby-version
+Gemfile.lock
```

Comment:
My bad, I usually include it in my modules, I didn't see that I didn't put it there this time. :)